tests/test_readers.py

```diff
@@ -0,0 +1,261 @@
+# This file is part of daf_butler.
```

Comment:
Not part of daf_butler.

Reply:
Ha!  Pretty clear where I'm stealing my new copyright stanzas.